infra/cifuzz/cifuzz.py

```diff
@@ -203,6 +209,40 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   return True, False
 
 
+def check_fuzzer_build(out_dir):
+  """Checks the integrity of the built fuzzers.
+
+  Args:
+    out_dir: The location of the fuzzer binaries.
+
+  Returns:
+    True if fuzzers are correct or False if not.
+  """
+  if not os.path.exists(out_dir):
+    logging.error('Invalid out directory: %s.', out_dir)
+    return False
+  if not os.listdir(out_dir):
+    logging.error('No fuzzers found in out directory: %s.', out_dir)
+    return False
+
+  command = [
+      '--cap-add', 'SYS_PTRACE', '-e', 'FUZZING_ENGINE=' + DEFAULT_ENGINE, '-e',
```

Comment:
if formatting allows it, I'd put each `'-e'` on the same line as the env variable it is setting.

Reply:
The formatter is not happy with the change.